README.md

```diff
@@ -3,7 +3,7 @@
 # Decentralized Web Node (DWN) SDK
 
 Code Coverage
-![Statements](https://img.shields.io/badge/statements-94.22%25-brightgreen.svg?style=flat) ![Branches](https://img.shields.io/badge/branches-93.94%25-brightgreen.svg?style=flat) ![Functions](https://img.shields.io/badge/functions-92.02%25-brightgreen.svg?style=flat) ![Lines](https://img.shields.io/badge/lines-94.22%25-brightgreen.svg?style=flat)
+![Statements](https://img.shields.io/badge/statements-94.21%25-brightgreen.svg?style=flat) ![Branches](https://img.shields.io/badge/branches-93.91%25-brightgreen.svg?style=flat) ![Functions](https://img.shields.io/badge/functions-92%25-brightgreen.svg?style=flat) ![Lines](https://img.shields.io/badge/lines-94.21%25-brightgreen.svg?style=flat)
```

Comment:
I think the main driver of test coverage drop is that `utils/protocols.ts` had 100% coverage and I've now removed it. Going to look at it again today to see if there's another culprit.